### ISSUE TITLE
Still failing at PyQt6 in 0.2.6 and Python 3.9

### ISSUE THREAD

**Issue by DeutscheGabanna** [debanjum/khoj#146]
```
python -m pip install khoj-assistant==0.2.6
Collecting khoj-assistant==0.2.6
  Downloading khoj_assistant-0.2.6-py3-none-any.whl (1.0 MB)
     ---------------------------------------- 1.0/1.0 MB 16.4 MB/s eta 0:00:00
Collecting pydantic==1.9.1
  Using cached pydantic-1.9.1-py3-none-any.whl (142 kB)
Collecting pillow==9.3.0
  Using cached Pillow-9.3.0-cp39-cp39-win32.whl (2.2 MB)
Collecting transformers==4.21.0
  Using cached transformers-4.21.0-py3-none-any.whl (4.7 MB)
Collecting schedule==1.1.0
  Using cached schedule-1.1.0-py2.py3-none-any.whl (10 kB)
Collecting fastapi==0.77.1
  Using cached fastapi-0.77.1-py3-none-any.whl (54 kB)
Collecting defusedxml==0.7.1
  Using cached defusedxml-0.7.1-py2.py3-none-any.whl (25 kB)
Collecting numpy==1.22.4
  Using cached numpy-1.22.4-cp39-cp39-win32.whl (12.2 MB)
Collecting openai==0.20.0
  Using cached openai-0.20.0.tar.gz (42 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting pyqt6==6.3.1
  Using cached PyQt6-6.3.1.tar.gz (1.0 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [22 lines of output]
      Traceback (most recent call last):
        File "C:\Users\Paul\AppData\Local\Programs\Python\Python39-32\lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
          main()
        File "C:\Users\Paul\AppData\Local\Programs\Python\Python39-32\lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "C:\Users\Paul\AppData\Local\Programs\Python\Python39-32\lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 152, in prepare_metadata_for_build_wheel
          whl_basename = backend.build_wheel(metadata_directory, config_settings)
        File "C:\Users\Paul\AppData\Local\Temp\pip-build-env-pwi35v7h\overlay\Lib\site-packages\sipbuild\api.py", line 46, in build_wheel
          project = AbstractProject.bootstrap('wheel',
        File "C:\Users\Paul\AppData\Local\Temp\pip-build-env-pwi35v7h\overlay\Lib\site-packages\sipbuild\abstract_project.py", line 87, in bootstrap
          project.setup(pyproject, tool, tool_description)
        File "C:\Users\Paul\AppData\Local\Temp\pip-build-env-pwi35v7h\overlay\Lib\site-packages\sipbuild\project.py", line 585, in setup
          self.apply_user_defaults(tool)
        File "project.py", line 67, in apply_user_defaults
          super().apply_user_defaults(tool)
        File "C:\Users\Paul\AppData\Local\Temp\pip-build-env-pwi35v7h\overlay\Lib\site-packages\pyqtbuild\project.py", line 70, in apply_user_defaults
          super().apply_user_defaults(tool)
        File "C:\Users\Paul\AppData\Local\Temp\pip-build-env-pwi35v7h\overlay\Lib\site-packages\sipbuild\project.py", line 236, in apply_user_defaults
          self.builder.apply_user_defaults(tool)
        File "C:\Users\Paul\AppData\Local\Temp\pip-build-env-pwi35v7h\overlay\Lib\site-packages\pyqtbuild\builder.py", line 69, in apply_user_defaults
          raise PyProjectOptionException('qmake',
      sipbuild.pyproject.PyProjectOptionException
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

C:\Users\Paul>where python
C:\Users\Paul\AppData\Local\Programs\Python\Python39-32\python.exe

C:\Users\Paul>
```

I followed all the advice I could get from both this repository as well as PyQt6 help channels.

- I ran `python -m pip install` instead just `pip install` to make sure I invoke the correct Python version
- I only have python 3.9 so whatever, actually, but I was told 3.9 is the version where khoji actually works
- I ran `where python` to also make sure I launch from python 3.9
- I used the latest version of khoji==0.2.6

**Comment by DeutscheGabanna** [comment]
So it seems that the #145 error has not been fixed yet.

**Comment by debanjum** [comment]
Hi @DeutscheGabanna, thanks for looking into this. As you noticed, the PyQT issue has not been fixed as of release 0.2.6. Issue #145 is meant to track the PyQT investigation and fix. 

It'd be great if you can put your investigation notes in #145 and close this issue. This will avoid duplicate issues tracking the same problem.
